clients/deltalake/src/test/java/org/projectnessie/deltalake/ITDeltaLogBranches.java

```diff
@@ -58,23 +60,23 @@ protected static void createDelta() {
   }
 
   @BeforeEach
-  public void createClient() {
-    client = NessieClient.builder().withUri(url).build();
+  void createClient() {
+    api = HttpClientBuilder.builder().withUri(url).build(NessieApiVersion.V_1, NessieApiV1.class);
   }
 
   @AfterEach
-  public void closeClient() throws NessieNotFoundException, NessieConflictException {
+  void closeClient() throws NessieNotFoundException, NessieConflictException {
     Reference ref = null;
     try {
-      ref = client.getTreeApi().getReferenceByName("test");
+      ref = api.getReference().refName("test").submit();
```

Comment:
do we need to do the same cleanup stuff in the other classes?

Reply:
Don't think so